styles.css

```diff
@@ -0,0 +1,106 @@
+* {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  outline: 0;
+  font-size: 100%;
+  vertical-align: baseline;
+  background: transparent;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'Inter', sans-serif;
+  color: #0a0908;
+  background: #fbf8f2;
+  text-transform: uppercase;
+  font-size: 11px;
+  font-weight: 600;
+}
+
+header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  height: 50px;
+  padding-right: 42px;
+}
+
+input[type="file"] {
+  display: none;
+}
+
+.image-input {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-shrink: 0;
+  margin-right: 32px;
+  border-right: 1px solid #0a0908;
+  padding: 24px;
+  cursor: pointer;
+  height: 50px;
+  background-color: transparent;
+  transition: all 200ms ease-in-out;
+}
+
+.image-input:hover {
+  background-color: #0a0908;
+  color: #fbf8f2;
+}
+
+.dg.ac {
```

Comment:
When working with a 3rd party module that has no style inputs the CSS can get pretty ugly, see below. Lots of `!important` which isn't best practice, but I'm sorry